internal/payloads/ami_payload.go

```diff
@@ -0,0 +1,48 @@
+package payloads
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/RHEnVision/provisioning-backend/internal/helpers"
+)
+
+type AmiResponse struct {
+	Distribution  string          `json:"distribution"`
+	ImageName     string          `json:"image_name"`
+	ImageRequests []ImageRequests `json:"image_requests"`
+}
+type Options struct {
+	ShareWithAccounts []string `json:"share_with_accounts"`
+}
+type UploadRequest struct {
+	Type    string  `json:"type"`
+	Options Options `json:"options"`
+}
+type ImageRequests struct {
+	Architecture  string        `json:"architecture"`
+	ImageType     string        `json:"image_type"`
+	UploadRequest UploadRequest `json:"upload_request"`
+}
+
+func CreateAMIrequest() string {
+	data := AmiResponse{
+		Distribution: "rhel-84",
+		ImageName:    "test-image",
+		ImageRequests: []ImageRequests{
+			{
+				Architecture: "x86_64",
+				ImageType:    "aws",
+				UploadRequest: UploadRequest{
+					Type: "aws",
+					Options: Options{
+						ShareWithAccounts: []string{os.Getenv("ACCOUNT_ID")},
+					},
+				},
+			},
+		},
+	}
+	payloadBytes, err := json.Marshal(data)
+	helpers.PanicOn(err)
```

Comment:
Let's avoid these "generic error handler helpers", the proper way to do this is really check for error via `if` and pass error as a return value from this function. Welcome to the Go world :)

Panic should only be used if you would create an API that will also call `defer` and then translate it to an error, which is not this case: https://go.dev/doc/effective_go#panic

Reply:
Yes It is a leftover I forgot to remove, I'll use the logger, but saw that I have cyclic import issue which I need to solve